tests/integration/test_vulnerability_detector/test_cpe_helper/test_cpe_indexing_empty_fields.py

```diff
@@ -0,0 +1,171 @@
+'''
+copyright: Copyright (C) 2015-2023, Wazuh Inc.
+
+           Created by Wazuh, Inc. <info@wazuh.com>.
+
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+type: integration
+
+brief: Wazuh is able to detect vulnerabilities in the applications installed in agents using the Vulnerability Detector
+       module. This software audit is performed through the integration of vulnerability feeds indexed by Redhat,
+       Canonical, Debian, Amazon Linux and NVD Database.
+
+components:
+    - vulnerability_detector
+
+suite: feeds
+
+targets:
+    - manager
+
+daemons:
+    - wazuh-modulesd
+    - wazuh-db
+
+os_platform:
+    - linux
+
+os_version:
+    - Arch Linux
+    - Amazon Linux 2
+    - Amazon Linux 1
+    - CentOS 8
+    - CentOS 7
+    - Debian Buster
+    - Red Hat 8
+    - Ubuntu Focal
+    - Ubuntu Bionic
+
+references:
+    - https://documentation.wazuh.com/current/user-manual/capabilities/vulnerability-detection/index.html
+
+tags:
+    - vulnerability_detector
+    - cpe_helper
+'''
+import os
+import json
+import pytest
+
+from wazuh_testing.constants.paths.logs import WAZUH_LOG_PATH
+from wazuh_testing.utils.configuration import (load_configuration_template, get_test_cases_data,
+                                        update_configuration_template)
+from wazuh_testing.utils.callbacks import generate_callback
+from wazuh_testing.modules.vulnerability_detector import patterns as cb
+from wazuh_testing.modules import vulnerability_detector as vd
+from wazuh_testing.modules.vulnerability_detector.packages import VULNERABLE_PACKAGES_EMPTY_VENDOR
+from wazuh_testing.tools.file_monitor import FileMonitor
+from . import TEST_CASES_PATH, CONFIGURATIONS_PATH, custom_nvd_json_feed_path
+
+local_internal_options = vd.LOCAL_INTERNAL_OPTIONS
```

Comment:
Fix this after the changes in the framework here and in all tests.

Reply:
Done in https://github.com/wazuh/wazuh/pull/17576/commits/d2f3b68b5947526a287450a641ec45704024eac4